src/react-components/chat-message.js

```diff
@@ -48,8 +48,11 @@ function renderChatMessage(body, from, allowEmojiRender) {
   el.setAttribute("class", `${styles.presenceLog} ${styles.presenceLogSpawn}`);
   document.body.appendChild(el);
 
-  const entryDom = (
+  const EntryDom = ({ callback }) => (
     <div
+      // callback is passed in here as part of React 18 createRoot method.
+      // eslint-disable-next-line react/no-unknown-property
+      callback={callback}
```

Comment:
createRoot and render methods for React 18 now require previous render callback argument to be passed in this way